dev/get_changes.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+#
+# Copyright 2018-2023 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+
+from github import Github
+import rich_click as click
```

Comment:
Please add requirements.txt if you're importing external libraries

Reply:
Thanks @mobuchowski . The requirements.txt in dev has been updated.